hypertrace-trace-enricher/helm/templates/trace-enricher-config.yaml

```diff
@@ -37,6 +37,9 @@ data:
     }
 
     enricher {
+      executors = {
+        thread.pool.size = {{ .Values.traceEnricherConfig.enricherExecutorsThreadPoolSize }}
```

Comment:
Based on my other comment. Wrap this in a `if`. So by default it remains the num of stream threads and we just provide an option to override if needed.